File: location-tracker-api/src/main/java/com/gmail/kadoshnikovkirill/locationtracker/dto/TrackDto.java
package com.gmail.kadoshnikovkirill.locationtracker.dto;

import lombok.AccessLevel;
import lombok.Builder;
import lombok.Getter;
import lombok.RequiredArgsConstructor;

import java.time.LocalDateTime;

@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
@Builder(toBuilder = true)
@Getter
public class TrackDto {

    private final Long userId;
    private final Float lat;
    private final Float lon;
    private final LocalDateTime timestamp;
    private final String countryCode;
    private final Integer postalCode;
    private final String country;
    private final String region;
    private final String city;
    private final String street;
    private final String house;

}

File: location-tracker-api/src/main/java/com/gmail/kadoshnikovkirill/locationtracker/dto/TrackDTO.kt
package com.gmail.kadoshnikovkirill.locationtracker.dto

import java.time.LocalDateTime

data class TrackDTO(
    val userId: Long,
    val lat: Float,
    val lon: Float,
    val timestamp: LocalDateTime,
    val countryCode: String?,
    val postalCode: Int?,
    val country: String?,
    val region: String?,
    val city: String?,
    val street: String?,
    val house: String?)